components/modal/config/ModalStatus.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace AntDesign
 {
-    internal enum ModalStatus
```

Comment:
Should it need to change the access modifier?

Reply:
@ElderJames A previous issue mentioned that it was hoped to get the status of the Modal externally, so this modification was made.